deployment/chainloop/values.yaml

```diff
@@ -144,6 +144,10 @@ controlplane:
   #    - name: "read-only-demo"
   #      role: "MEMBERSHIP_ROLE_ORG_VIEWER"
 
+  ## @extra controlplane.prometheus_integration.org_name Name of the organization to integrate with Prometheus
```

Comment:
this documentation doesnt seem right. org_name is a property of an object in the array.

I'd just document the parent one 

> @extra controlplane.prometheus_integration Expose prometheus metrics for your organization